src/store/actions/authActions.js

```diff
@@ -14,7 +14,7 @@ const loginAction = (user) => {
       return session
         .begin(user)
         .then((whoami) => {
-          if (whoami?.username !== process.env.REACT_APP_ANONYMOUS_USER) {
+          if (!whoami?.anonymous) {
```

Comment:
Note to myself..:

What if whoami is undefined?

Reply:
![image](https://user-images.githubusercontent.com/52858351/89473351-ebcca880-d782-11ea-8144-d7837b8f2801.png)

This is issue is not urgent cause whoami cannot be undefined.
However we should adjust.